qiskit_optimization/applications/ising/vehicle_routing.py

```diff
@@ -10,187 +10,215 @@
-
-    c_z = c_z + np.trace(q_z)
-    q_z = q_z - np.diag(np.diag(q_z))
-
-    # Getting the Hamiltonian in the form of a list of Pauli terms
-
-    pauli_list = []
-    for i in range(N):
-        if g_z[i] != 0:
-            w_p = np.zeros(N)
-            v_p = np.zeros(N)
-            v_p[i] = 1
-            pauli_list.append((g_z[i], Pauli((v_p, w_p))))
-    for i in range(N):
-        for j in range(i):
-            if q_z[i, j] != 0:
-                w_p = np.zeros(N)
-                v_p = np.zeros(N)
-                v_p[i] = 1
-                v_p[j] = 1
-                pauli_list.append((2 * q_z[i, j], Pauli((v_p, w_p))))
-
-    pauli_list.append((c_z, Pauli((np.zeros(N), np.zeros(N)))))
-    opflow_list = [(pauli[1].to_label(), pauli[0]) for pauli in pauli_list]
-    return PauliSumOp.from_list(opflow_list)
-
-
-def get_vehiclerouting_solution(instance: np.ndarray,
-                                n: int,
-                                K: int,
-                                result: MinimumEigensolverResult) -> List[int]:
-    """Tries to obtain a feasible solution (in vector form) of an instance
-        of vehicle routing from the results dictionary.
-
-    Args:
-        instance: a customers-to-customers distance matrix.
-        n: the number of customers.
-        K: the number of vehicles available.
-        result: a result obtained by QAOA.run or VQE.run.
-
-    Returns:
-        a solution, i.e., a path, in its binary representation.
-
-    #TODO: support statevector simulation, results should be a statevector or counts format, not
-           a result from algorithm run
-    """
-    del instance, K  # unused
-    v = result.eigenstate
-    N = (n - 1) * n
-
-    index_value = [x for x in range(len(v)) if v[x] == max(v)][0]
-    string_value = "{0:b}".format(index_value)
 
-    while len(string_value) < N:
-        string_value = '0' + string_value
-
-    x_sol = list()
-    for elements in string_value:
-        if elements == '0':
-            x_sol.append(0)
+    def __init__(self, graph: Union[nx.Graph, np.ndarray, List],
+                 num_vehicles: int = 2, depot: int = 0) -> None:
+        """
+        Args:
+            graph: A graph of the NetworkX. It can be anything that the constructor of
+            networkx.Graph can accept.
+            num_vehicles: The number of vehicles
+            depot: The index of the depot node where all the vehicle depart
+        """
+        super().__init__(graph)
+        self._num_vehicles = num_vehicles
+        self._depot = depot
+
+    def to_quadratic_program(self) -> QuadraticProgram:
+        """Convert a vehicle routing problem instance into a
+        :class:`~qiskit_optimization.problems.QuadraticProgram`
+
+        Returns:
+            The :class:`~qiskit_optimization.problems.QuadraticProgram` created
+            from the vehicle routing problem instance.
+        """
+        mdl = Model(name='Vehicle Routing')
```

Comment:
I note in this name both words are capitalized, whereas it was `Stable set` above for that one, where only the first work was capitalized. Its a minor comment, I just noticed the difference that was all.